app/models/assignment_participant.rb

```diff
@@ -84,33 +84,32 @@ def includes?(participant)
   end
 
   def assign_reviewer(reviewer)
-    ParticipantReviewResponseMap.create(:reviewee_id => self.id, :reviewer_id => reviewer.id,
-                                        :reviewed_object_id => assignment.id)
+    ParticipantReviewResponseMap.create(reviewee_id: self.id, reviewer_id: reviewer.id,
+                                        reviewed_object_id: assignment.id)
   end
 
   def assign_quiz(contributor,reviewer,topic)
-    participant_id=AssignmentParticipant.where(topic_id: topic, parent_id:  contributor.parent_id).first.id
+    participant_id=AssignmentParticipant.where(topic_id: topic, parent_id: contributor.parent_id).first.id
 
-    quiz = QuizQuestionnaire.find_by_instructor_id(contributor.id)
```

Comment:
I changed the .find_by_instructor_id to .where(instructor_id:) according to the global rules.